.github/workflows/release.yml

```diff
@@ -1,118 +1,22 @@
-        with:
-          java-version: 11
-          distribution: 'temurin'
-          cache: 'maven'
-          server-id: ossrh
-          server-username: MAVEN_USERNAME
-          server-password: MAVEN_PASSWORD
-          gpg-private-key: ${{ secrets.MAVEN_GPG_KEY }}
-          gpg-passphrase: MAVEN_GPG_PASSPHRASE
-
-      - name: Publish SNAPSHOT
-        run: mvn -B --no-transfer-progress clean deploy
-        env:
-          MAVEN_USERNAME: ${{ secrets.OSSRH_USERNAME }}
-          MAVEN_PASSWORD: ${{ secrets.OSSRH_PASSWORD }}
-          MAVEN_GPG_PASSPHRASE: ${{ secrets.MAVEN_GPG_PASSPHRASE }}
-
-  # Run for manual trigger (workflow dispatch), since you'll have release and next dev versions specified
-  # All commits will have a -SNAPSHOT project version anyway, since the releases will be handled here
-  release:
-    needs: setup
-    if: github.event_name == 'workflow_dispatch'
-    runs-on: ubuntu-latest
-    env:
-      RELEASE: ${{ inputs.releaseversion }}
-      NEXT: ${{ inputs.nextversion }}
-    steps:
-      - name: Checkout latest code
-        uses: actions/checkout@v3
-
-      - name: Config git user
+      - name: Check users available
         run: |
-          git config user.name ${{ github.actor }}
-          git config user.email "${{ github.actor }}@users.noreply.github.com"
+          echo "Using OSSRH_USERNAME ${{ secrets.OSSRH_USERNAME }}"
+          echo "Using GH_USER ${{ secrets.GH_USER }}"
 
-      - name: Setup Java & Maven
-        uses: actions/setup-java@v3
-        with:
-          java-version: 11
-          distribution: 'temurin'
-          cache: 'maven'
-          server-id: ossrh
-          server-username: MAVEN_USERNAME
-          server-password: MAVEN_PASSWORD
-          gpg-private-key: ${{ secrets.MAVEN_GPG_KEY }}
-          gpg-passphrase: MAVEN_GPG_PASSPHRASE
-      # =============================================================================
-      # Start the release
-      # =============================================================================
-      - name: Release main POM
+      - name: DRYRUN -- Release main POM
         run: |
-          mvn -B -U -V -ntp release:prepare -DreleaseVersion=$RELEASE -Dtag=$RELEASE -DdevelopmentVersion=$NEXT -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn
-          mvn -B -U -V -ntp release:perform -P release -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn
-        env:
-          MAVEN_USERNAME: ${{ secrets.OSSRH_USERNAME }}
-          MAVEN_PASSWORD: ${{ secrets.OSSRH_PASSWORD }}
-          MAVEN_GPG_PASSPHRASE: ${{ secrets.MAVEN_GPG_PASSPHRASE }}
-
-      - name: Build and publish new dev version
-        run: mvn -B -U -V -ntp deploy -P release
+          mvn -B -U -V -ntp release:prepare -DdryRun=true -DreleaseVersion=0.001 -Dtag=0.001 -DdevelopmentVersion=0.002 -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn
```

Comment:
may be able to use this Maven command to test things out:

```
mvn -B -V -ntp verify gpg:sign -P release
```

The release profile used here should trigger the `gpg:sign` goal anyway, but it won't hurt anything to be explicit about his goal anyway.

I haven't been able to get this to error out locally, so I'm not 100% sure this will do the trick